pulsebot/pulse_hgpushes.py

```diff
@@ -59,11 +63,12 @@ def get_pushes_info(pulse_message):
     def get_push_info_from(push_url):
         repo = push_url[: push_url.rindex("/")]
         r = requests.get(push_url)
-        if r.status_code == 500:
-            # If we got an error 500, try again once.
+        if r.status_code != requests.codes.ok:
+            # If we were not successful, try again once.
             r = requests.get(push_url)
-        if r.status_code != 200:
-            r.raise_for_status()
+        if r.status_code != requests.codes.ok:
+            logger.exception(f"Failure getting {push_url} {r.status_code}")
```

Comment:
```suggestion
            logger.error(f"Failure getting {push_url} {r.status_code}")
```

Would be good to add a `logger.info` here saying that we are trying again.